jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -136,7 +136,7 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 false,
                 message("codemodernizer.notification.warn.invalid_project.description.reason.missing_content_roots"),
                 InvalidTelemetryReason(
-                    CodeTransformPreValidationError.EmptyProject
+                    CodeTransformPreValidationError.NoPom
```

Comment:
Is it possible to keep the reason more generic, such as NoBuildFile? Q Transformation will support other buildTools like Gradle and "missing_content_roots" is applicable to these other buildTools.

Reply:
Right now its not clear what we are going to support. And right now specifics around maven/gradle are useful metrics to have to determine how many projects we have or do not have with our specific requirements.